Mini Portal Redesign

steets250 opened the pull request.

steets250 commented:
Ok time to review this bad boy.

StormFireFox1 reviewed (changes requested):
Looks good!

Maybe add that additional wack font for the ACM logo.

leestanley commented:
Ok slight nitpick but can we add a hover effect as a mixin and apply it to all the buttons for auth flow

annsudhart commented on src/components/PasswordForm/style.less:
```diff
@@ -1,49 +1,50 @@
-    width: 200px;
+    .input-box {
+      border: none;
+      border-bottom: 1px solid @black;
```
The login input element's font size seems a bit too small, it looks better with `1rem` instead of `1em`

![image](https://user-images.githubusercontent.com/32887362/95108481-2bb2fc00-06f0-11eb-9ab9-db0278d95779.png)

annsudhart commented on src/components/Header/index.tsx:
```diff
@@ -0,0 +1,31 @@
+        <div className="header">
+          <NavLink className="title" to="/">
+            <img className="logo" src={logo} alt="ACM Logo" />
+            <span className="subheading">&nbsp;Membership Portal</span>
```
The top bar text and acm logo are too close together, maybe add `margin: 0 1em` to the subheading class style

![Uploading image.png…]()

steets250 commented:
@leestanley : I added the button mixin to the sign in, sign up, password reset, and password change forms.
@annsudhart I added padding to the site header and bumped up the input sizing to 16px.
@StormFireFox1 : I talked with design, and we agreed to just stick with DM Sans for now.

StormFireFox1 reviewed (approved):
Kickoff merge when?
